frontend/src/toolbar/elements/ElementInfoWindow.tsx

```diff
@@ -65,59 +65,41 @@ export function ElementInfoWindow(): JSX.Element | null {
 
     return (
         <div
+            className="absolute z-[1] opacity-100 origin-top-left transition-opacity transition-shadow duration-200 bg-white"
             // eslint-disable-next-line react/forbid-dom-props
             style={{
                 pointerEvents: pointerEvents ? 'all' : 'none',
-                position: 'absolute',
                 top,
                 bottom,
                 left,
                 width,
                 minHeight,
                 maxHeight,
-                zIndex: 1,
-                opacity: 1,
-                transformOrigin: 'top left',
-                transition: 'opacity 0.2s, box-shadow 0.2s',
                 backgroundBlendMode: 'multiply',
-                background: 'white',
             }}
         >
             {onClose ? (
                 <div
                     onClick={onClose}
+                    className="absolute origin-top-left bg-black text-white rounded-full w-6 h-6 z-[7] flex items-center justify-around text-center cursor-pointer"
```

Comment:
I don't think we have a color var `black` defined in the `$colors` list of vars.scss – probably should be:
```suggestion
                    className="absolute origin-top-left bg-[#000] text-white rounded-full w-6 h-6 z-[7] flex items-center justify-around text-center cursor-pointer"
```